.circleci/ubuntu1804-python2.dockerfile

```diff
@@ -0,0 +1,90 @@
+FROM ubuntu:18.04
+
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && \
+    apt-get install -y \
+        ca-certificates \
+        chromium-browser \
+        curl \
+        expect \
+        expect-dev \
+        gdb \
+        gir1.2-gstreamer-1.0 \
+        gir1.2-gudev-1.0 \
+        git \
+        gstreamer1.0-libav \
+        gstreamer1.0-plugins-bad \
+        gstreamer1.0-plugins-base \
+        gstreamer1.0-plugins-good \
+        gstreamer1.0-tools \
+        gstreamer1.0-x \
+        gzip \
+        language-pack-en \
+        libgstreamer1.0-dev \
+        libgstreamer-plugins-base1.0-dev \
+        libopencv-dev \
+        liborc-0.4-dev \
+        librsvg2-bin \
+        lighttpd \
+        moreutils \
+        pep8 \
+        python-dev \
+        python-docutils \
+        python-flask \
+        python-gi \
+        python-jinja2 \
+        python-kitchen \
+        python-libcec \
+        python-lxml \
+        python-matplotlib \
+        python-mock \
+        python-nose \
+        python-numpy \
+        python-opencv \
+        python-pip \
+        python-pysnmp4 \
+        python-qrcode \
+        python-requests \
+        python-scipy \
+        python-serial \
+        python-yaml \
+        python-zbar \
+        ratpoison \
+        socat \
+        ssh \
+        sudo \
+        tar \
+        tesseract-ocr \
+        tesseract-ocr-deu \
+        tesseract-ocr-eng \
+        time \
+        v4l-utils \
+        wget \
+        xdotool \
+        xserver-xorg-video-dummy \
+        xterm && \
+    apt-get clean
+
+RUN pip install \
+        astroid==1.4.8 \
+        isort==3.9.0 \
+        pylint==1.6.4 \
```

Comment:
18.04 comes with pylint 1.8.3, asteroid 1.6.0 and pytest 3.3.2 in apt.

I see you fix pytest in later commit, but we still install pylint from pip.  What's the rationale?

Reply:
> we still install pylint from pip. What's the rationale?

So that `make check-lint` works the same on Ubuntu 16.04 & 18.04. It has been difficult in the past to write code that satisfies different versions of pylint.

On the other hand It'd be worth ensuring that **stbt-lint** works with both Ubuntu 16.04 & 18.04's versions of pylint.